Misc/NEWS.d/next/Documentation/2018-06-20-10-22-57.bpo-33783.9MeA9Q.rst

```diff
@@ -0,0 +1 @@
+Use a proper Sphinx class directive for :class:`random.Random`
```

Comment:
We don't require a NEWS entry for small documentation patches.

Reply:
I know, but then `bedevere/news`  mark the test as failing, in the all-PR view of GitHub the test appear as failing and then the PRs tends to not be reviewed... removed.